resource_manager/google/cloud/resource_manager/project.py

```diff
@@ -202,7 +205,16 @@ def update(self, client=None):
         """
         client = self._require_client(client)
 
-        data = {'name': self.name, 'labels': self.labels}
+        data = {}
```

Comment:
Is it required that the keys not be set if there's no data for the request?
If you can set all the keys, then you could do something like this and drop the `if`s.

```python
data {
    'name': self.name,
    'labels': self.labels,
    'parent': self.parent,
}
```